src/components/tree/tree.tsx

```diff
@@ -110,7 +110,9 @@ export const Tree = withStyles(styles)(
                             <i onClick={() => this.props.toggleItemOpen(it.id, it.status)}
                                 className={toggableIconContainer}>
                                 <ListItemIcon className={this.getToggableIconClassNames(it.open, it.active)}>
-                                    {it.status !== TreeItemStatus.INITIAL && it.items && it.items.length === 0 ? <span /> : <SidePanelRightArrowIcon />}
+                                    {it.status === TreeItemStatus.PENDING 
+                                    || (it.status === TreeItemStatus.LOADED && !it.items)
+                                    || (it.status === TreeItemStatus.LOADED && it.items && it.items.length === 0) ? <span /> : <SidePanelRightArrowIcon />}
                                 </ListItemIcon>
```

Comment:
Quite complex condition. It should be extracted to local function. Btw I guess `it.items` is always set so checking if it exists is not needed?